deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -179,7 +179,6 @@ data:
         cache_size "#{ENV['K8S_METADATA_FILTER_BEARER_CACHE_SIZE']}"
         cache_ttl "#{ENV['K8S_METADATA_FILTER_BEARER_CACHE_TTL']}"
         tag_to_kubernetes_name_regexp '.+?\.containers\.(?<pod_name>[^_]+)_(?<namespace>[^_]+)_(?<container_name>.+)-(?<docker_id>[a-z0-9]{64})\.log$'
-        merge_json_log false
```

Comment:
Why do need to remove this one?

Reply:
This parameter was removed in the newer version of `kubernets_metadata` plugin, from their README:

> NOTE: As of the release 2.1.x of this plugin, it no longer supports parsing the source message into JSON and attaching it to the payload. The following configuration options are removed:

> merge_json_log
> preserve_json_log